JSDemos/Demos/List/Item3RdPartyEngineTemplate/jQuery/index.html

```diff
@@ -8,7 +8,7 @@
     <script src="../../../../../node_modules/jquery/dist/jquery.min.js"></script>
     <link rel="stylesheet" type="text/css" href="../../../../../node_modules/devextreme/dist/css/dx.light.css" />
     <script src="../../../../../node_modules/devextreme/dist/js/dx.all.js"></script>
-    <script src="../../../../../node_modules/underscore/underscore.js"></script>
+    <script src="../../../../../node_modules/mustache/mustache.min.js"></script>
```

Comment:
Some demos (especially this one) have own descriptions about the 3-rd party engines:

https://github.com/DevExpress/devextreme-demos/blob/23_1/JSDemos/Demos/List/Item3RdPartyEngineTemplate/description.md

Should we edit them as well? @dxvladislavvolkov @VasilyStrelyaev

Reply:
Yes. We need to edit it